apps/web/app/future/availability/page.tsx

```diff
@@ -10,4 +14,20 @@ export const generateMetadata = async () => {
   );
 };
 
-export default WithLayout({ getLayout: null, Page: AvailabilityPage });
+const Page = async () => {
+  const session = await getServerSession(AUTH_OPTIONS);
+  const userId = session?.user?.id ?? -1;
+  const orgId = session?.user?.org?.id ?? -1;
+
+  let currentOrg = null;
+  try {
```

Comment:
I don't think we should we should make the query for the current org if no orgId is present.